054-infinite-auto-restart-of-Kafka-connectors.md

```diff
@@ -0,0 +1,160 @@
+# Infinite auto-restart of Apache Kafka connectors
+
+This proposal introduces changes to the connector auto-restart functionality which was introduced in [Proposal #7](https://github.com/strimzi/proposals/blob/main/007-restarting-kafka-connect-connectors-and-tasks.md) and implemented in [Strimzi 0.33.0](https://github.com/strimzi/strimzi-kafka-operator/pull/7500).
+
+## Current situation
+
+Strimzi 0.33.0 introduced support for automatically restarting connectors in Apache Kafka Connect clusters (when managed using the `KafkaConnector` resources). This feature is disabled by default, but can be enabled in the `KafkaConnectors` `.spec` section:
+
+```yaml
+apiVersion: strimzi.io/v1beta2
+kind: KafkaConnector
+metadata:
+  name: my-source-connector
+spec:
+  # ...
+  autoRestart:
+    enabled: false
```

Comment:
Should this be true to show how to enable it (because disabled by default as you mentioned above)?